src/styles/calendar.scss

```
@@ -0,0 +1,52 @@
+.calendar-container {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+}
+
+.calendar {
+  display: flex;
+  flex-wrap: wrap;
+  gap: $gap;
+  padding: 10px;
+  max-width: $calendar-width;
+}
+
+.calendar__day {
```

As I said before use scss nesting, please 

```suggestion
  max-width: $calendar-width;

  &__day {
```